R/utility_fxns.R

```diff
@@ -0,0 +1,21 @@
+#' @export
+format_inputs <- function(nc_path){
+
+  assertthat::assert_that(file.exists(nc_path))
+
+  # Import the data and format it
+  nc <- ncdf4::nc_open(nc_path)
+
+  PAR <- ncdf4::ncvar_get(nc, "PAR")
+  for (i in which(PAR < -10)) {
+    PAR[i] <- PAR[i - 1]
```

Comment:
What about if i=1?

Reply:
ummm no idea this... this is pulled directly from their code...

Reply:
File under "not our problem" then